code/cron.yaml

```diff
@@ -0,0 +1,19 @@
+cron:
```

Comment:
I take it we were able to verify this works on the testing AppEngine instance? What manual testing did you perform there?

Reply:
Yes I tested it using our old project by having a simple server that would log every minute. Once that work I deployed it to our current project to see if it worked. Additionally, on the README there is a link to see the logs. Here it is: https://console.cloud.google.com/logs?_ga=2.119762766.714168666.1594744665-601550938.1590535110

Reply:
Awesome, thanks!